packages/api/src/utils/logs.js

```diff
@@ -237,28 +266,29 @@ export class Logging {
    * @param {string} name
    * @param {any} [description]
    */
-  time(name, description) {
+  time(name, description, context = {}) {
     this._times.set(name, {
       name: name,
       description: description,
       start: Date.now(),
+      context,
     })
     this._timesOrder.push(name)
   }
 
   /**
    * @param {string} name
+   * @param {boolean} [shouldLog]
+   * @param {any} [context]
    */
-  timeEnd(name) {
+  timeEnd(name, shouldLog, context) {
```

Comment:
This should match the [`timeEnd`](https://developer.mozilla.org/en-US/docs/Web/API/console/timeEnd) method.